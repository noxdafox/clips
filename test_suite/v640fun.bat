(clear)  ; print and println commands    
(println)
(print)
(println "Hello World")
(print "Hello World" crlf)                   
(println a " " "b" " " [c] " " 3 " " 4.5 " " (create$ d e f))
