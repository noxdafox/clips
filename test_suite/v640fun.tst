(unwatch all)
(clear)
(set-strategy depth)
(open "Results//v640fun.rsl" v640fun "w")
(dribble-on "Actual//v640fun.out")
(batch "v640fun.bat")
(dribble-off)
(load "compline.clp")
(printout v640fun "v640fun.bat differences are as follows:" crlf)
(compare-files "Expected//v640fun.out" "Actual//v640fun.out" v640fun)
; close result file
(close v640fun)
