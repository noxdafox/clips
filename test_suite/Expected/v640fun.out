TRUE
CLIPS> (batch "v640fun.bat")
TRUE
CLIPS> (clear)  ; print and println commands    
CLIPS> (println)

CLIPS> (print)
CLIPS> (println "Hello World")
Hello World
CLIPS> (print "Hello World" crlf)                   
Hello World
CLIPS> (println a " " "b" " " [c] " " 3 " " 4.5 " " (create$ d e f))
a b [c] 3 4.5 (d e f)
CLIPS> (dribble-off)
